.github/workflows/ci.yml

```diff
@@ -23,48 +23,48 @@ jobs:
       - name: Run linter
         run: make vet && make lint
 
-  test:
-    needs: lint
-    strategy:
-      matrix:
-        os: [ubuntu-latest, macos-latest]
-        go: ["1.22"]
-    name: ${{ matrix.os }} @ Go ${{ matrix.go }}
-    runs-on: ${{ matrix.os }}
-    env:
-      GO111MODULE: on
-      GOPROXY: https://proxy.golang.org
-    steps:
-      - name: Set up Go ${{ matrix.go }}
-        uses: actions/setup-go@v5
-        with:
-          go-version: ${{ matrix.go }}
-          cache: false
+  # test:
+  #   needs: lint
+  #   strategy:
+  #     matrix:
+  #       os: [ubuntu-latest, macos-latest]
+  #       go: ["1.22"]
+  #   name: ${{ matrix.os }} @ Go ${{ matrix.go }}
+  #   runs-on: ${{ matrix.os }}
+  #   env:
+  #     GO111MODULE: on
+  #     GOPROXY: https://proxy.golang.org
+  #   steps:
+  #     - name: Set up Go ${{ matrix.go }}
+  #       uses: actions/setup-go@v5
+  #       with:
+  #         go-version: ${{ matrix.go }}
+  #         cache: false
 
-      - name: Checkout Code
-        uses: actions/checkout@v4
-        with:
-          ref: ${{ github.ref }}
+  #     - name: Checkout Code
+  #       uses: actions/checkout@v4
+  #       with:
+  #         ref: ${{ github.ref }}
 
-      - uses: actions/cache@v4
-        with:
-          path: |
-            ${{ matrix.go-build }}
-            ~/go/pkg/mod
-          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-go-
+  #     - uses: actions/cache@v4
+  #       with:
+  #         path: |
+  #           ${{ matrix.go-build }}
+  #           ~/go/pkg/mod
+  #         key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+  #         restore-keys: |
+  #           ${{ runner.os }}-go-
 
-      - name: Run Tests
-        run: make test-coverage
+  #     - name: Run Tests
+  #       run: make test-coverage
 
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v4
-        with:
-          token: ${{ secrets.CODECOV_TOKEN }}
-          flags: ${{ matrix.os }},go-${{ matrix.go }}
-          slug: ${{ github.repository }}
+  #     - name: Upload coverage to Codecov
+  #       uses: codecov/codecov-action@v4
+  #       with:
+  #         token: ${{ secrets.CODECOV_TOKEN }}
+  #         flags: ${{ matrix.os }},go-${{ matrix.go }}
+  #         slug: ${{ github.repository }}
 
-      - name: Format
-        if: matrix.go-version == '1.22.x'
-        run: make fmt-check
+  #     - name: Format
+  #       if: matrix.go-version == '1.22.x'
+  #       run: make fmt-check
```

Comment:
- If you decide to remove the testing, please delete these lines of code directly instead of commenting them out. Commented-out code may lead to confusion, as it's not immediately clear why it's been disabled. Since we have version control, these lines are never lost.
- Please ensure that the format check is retained, as it serves a separate purpose from the testing.